internal/cluster/validator.go

```diff
@@ -524,3 +526,39 @@ func (v *clusterValidator) printNtpServerConfigured(c *clusterPreprocessContext,
 		return fmt.Sprintf("Unexpected status %s", status)
 	}
 }
+
+func (v *clusterValidator) isOcsRequirementsSatisfied(c *clusterPreprocessContext) validationStatus {
+	// check for all the operators here.
+
+	var operators models.Operators
+
+	if c.cluster.Cluster.Operators == "" {
+		return boolValue(true)
+	}
+
+	err := json.Unmarshal([]byte(c.cluster.Cluster.Operators), &operators)
+	if err != nil {
+		v.log.WithError(err).Warnf("Something went wrong with Unmarshalling operators")
+		return boolValue(false)
+	}
+
+	for _, operator := range operators {
+		if operator.OperatorType == models.OperatorTypeOcs && swag.BoolValue(operator.Enabled) == (true) {
+			return boolValue(v.ocsValidator.ValidateOCSRequirements(&c.cluster.Cluster))
+		}
+	}
+
+	return boolValue(true)
+
+}
+
+func (v *clusterValidator) printOcsRequirementsSatisfied(c *clusterPreprocessContext, status validationStatus) string {
+	switch status {
+	case ValidationSuccess:
+		return "OCS Requirements satisfied"
+	case ValidationFailure:
+		return "Insufficient resources on the cluster for deployment of the Ocs operator"
```

Comment:
I think that we should write more informative data for the user. I saw that you have it when you write it to the logs in ValidateOCSRequirements

Reply:
added